src/Symfony/Bundle/FrameworkBundle/Controller/Controller.php

```diff
@@ -124,6 +127,81 @@ protected function json($data, $status = 200, $headers = array(), $context = arr
     }
 
     /**
+     * Returns a BinaryFileResponse object with original or customized file name and disposition header.
+     *
+     * @param File|string $file        File object, path to file or string with content to be sent as response
+     * @param string|null $fileName    File name to be sent to response or null (will use original file name)
+     * @param string      $disposition Disposition of response (attachment is default, other type is inline)
```

Comment:
You could quote with `"` "attachment" and "inline" to improve the readability as keywords.